CONTRIBUTING.md

```diff
@@ -7,3 +7,6 @@ generative_data_prep follows standard [PEP8](https://peps.python.org/pep-0008/)
 ## Docstrings
 
 generative_data_prep uses [Google style docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) for formatting docstrings.
+
+## Formatting
+Please run `./format.sh` to format your code.
```

Comment:
Ah I did not have permissions to do that for some reason. Are there permissions tied to executing git files somewhere?